### DISCUSSION TITLE
Job Schedule via Spring dynamic proxy?

### DISCUSSION THREAD

**Discussion by albilu** [jobrunr/jobrunr#814] [Q&A] [answered]
Is it possible to schedule jobs via annotations like with the built in @Scheduled in spring?

I thought the Jobrunr @Job will do it but apparently not.

Motivation for that is to allow smoother integration of the Jobrunr library in existing spring projects using already @Scheduled.

It is more easier than create an additional service and try to find a place to register the Jobrunr Jobs

**Comment by rdehuyss** [comment]
Yes, it is in the docs.

**Reply by albilu** [reply]
sorry to bother. can you please share a link i found nothing in the docs. i am certainly looking in the wrong place

**Reply by rdehuyss** [reply, marked as answer]
On the recurring jobs page, search for spring.
